### ISSUE TITLE
VERBOSE: Pooling job . Status: .

### ISSUE THREAD

**Issue by afortaleza** [SitecorePowerShell/Console#743]
Steps to replicate the error:
* Install Sitecore v80r150812
* Install package **4.2 Minimal for Sitecore 8** into v80r150812
* Install powershell module **Remoting for SPE 4.2**
* Download **4.2 for Sitecore 8** to a local folder, say _C:\Download_
* Try to run the following script from Powershell ISE
```powershell
$session = New-ScriptSession -Username "admin" -Password "b" -ConnectionUri "http://v80r150812"
$jobId = Invoke-RemoteScript -Session $session -ScriptBlock {
    Install-Package -Path "C:\Download\Sitecore PowerShell Extensions-4.2 for Sitecore 8.zip"
}  -AsJob
Wait-RemoteScriptSession -Session $session -Id $jobId -Delay 5 -Verbose
```
In my environment I'm getting the following output and it never completes the job
> VERBOSE: Pooling jon . Status: .
> VERBOSE: Pooling jon . Status: .
> VERBOSE: Pooling jon . Status: .
> VERBOSE: Pooling jon . Status: .
> VERBOSE: Pooling jon . Status: .
> VERBOSE: Pooling jon . Status: .

**Comment by AdamNaj** [comment]
There is a known general issue with package installation if your instance does not have a MongoDB running while not in CMS only mode.

Can you verify if that is not the case?

**Comment by afortaleza** [comment]
Hello @AdamNaj, I have the following mongodb running:

> db version v3.2.10
> git version: 79d9b3ab5ce20f51c272b4411202710a082d0317
> OpenSSL version: OpenSSL 1.0.1t-fips  3 May 2016
> allocator: tcmalloc
> modules: none
> build environment:
>    distmod: 2008plus-ssl
>    distarch: x86_64
>    target_arch: x86_64

**Comment by afortaleza** [comment]
It works if I do the same procedure using:
* **4.1 Minimal for Sitecore 8**
* **Remoting for SPE 4.1**
* **SPE 4.1 for Sitecore 8**

**Comment by afortaleza** [comment]
Following some tests I believe the issue is with the **4.2 Minimal for Sitecore 8** package,

**Comment by michaellwest** [comment]
@afortaleza Do you happen to have the Remoting module installed in more than one location? You can check the `$env:PSModulePath` to see which locations are searched.

**Comment by afortaleza** [comment]
@michaellwest I have checked the 4 directories listed and it's installed in one directory only.

**Comment by michaellwest** [comment]
Hey @afortaleza, found the issue. Pretty silly actually. @AdamNaj and I just introduced a new security feature which requires the service to be enabled through configuration ("remoting=true") and the user to be added to the configured group ("sitecore\PowerShell Extensions Remoting"). 

If I connect with the user **sitecore\speremoting** and patch the *remoting* setting to *true* then the job starts working as expected. 

What would have been nice is that the **Wait-RemoteScriptSession** was aware no job id existed and return right away. In addition, it would have bee nice if the **New-ScriptSession** command reported that the service was disabled.

**Comment by michaellwest** [comment]
@afortaleza added commits for issues #752 and #753 to help address this issue.

**Comment by AdamNaj** [comment]
Assuming this is now a duplicate addressed by #752 and #753. Re-open otherwise.
